docs/install.rst

```diff
@@ -35,6 +35,22 @@ following line in your ``.zshrc`` after activating Antigen::
 
    antigen bundle nojhan/liquidprompt
 
+Installation via Zinit
+----------------------
+
+To install via `Zinit <https://github.com/zdharma/zinit>` _, simply add the following line in your ``.zshrc`` after activating Zinit::
```

Comment:
There should be no space between the '`' and '_', or the link does not work. You can see this in the compiled docs, linked in the build jobs.